src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -4749,6 +4742,18 @@ void handleInitializer(IOperation innerInitializer)
 
                     case OperationKind.Literal:
                         // PROTOTYPE(dataflow): See Microsoft.CodeAnalysis.VisualBasic.UnitTests.BindingErrorTests.BC30994ERR_AggrInitInvalidForObject
+                        //                      Just drop it for now to enable other test scenarios.
```

Comment:
Is it likely that we are missing on handling more such operations kinds in this switch statement? If so, should we consider changing the default clause to assert and return instead of throwing?

Reply:
> Is it likely that we are missing on handling more such operations kinds in this switch statement? If so, should we consider changing the default clause to assert and return instead of throwing?

This is likely what we will end up doing, even without an assert. However, we should look at the scenarios to make sure we are not missing something important